core/ir/instr_shared.c

```diff
@@ -470,6 +470,25 @@ instr_get_category(instr_t *instr)
 /* in rest of file, directly de-reference for performance (PR 622253) */
 #define instr_get_category inlined_instr_get_category
 
+const char *
+instr_get_category_name(dr_instr_category_t category)
```

Comment:
Please add the new API to the changelist at https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L195

Reply:
Done.